src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/IepSummary.kt

```diff
@@ -65,6 +65,8 @@ data class IepDetail(
   val id: Long? = null,
   @Schema(description = "IEP Level", example = "Standard", required = true)
   val iepLevel: String,
+  @Schema(description = "IEP Code", example = "STD", required = false)
```

Comment:
Made this optional because we also use `IepSummary` as the [return type from PrisonAPI](https://github.com/ministryofjustice/hmpps-incentives-api/blob/34319de9edf22b0a806125d739852df15edf40a8/src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonApiService.kt#L25-L39) and PrisonAPI does not provide it

I did think about making it required, but this would involve

1. Introducing a new type for PrisonAPI responses
2. Mapping the `iepLevel` returned to `iepCode`

Happy to discuss if folks think the alternative is better!